content/kubeflow/install.md

```diff
@@ -38,29 +44,36 @@ chmod +x aws-iam-authenticator
 sudo mv aws-iam-authenticator /usr/local/bin
 ```
 
-Set Kubeflow application name:
+Run **kfctl build** command to set up your configuraiton
 
 ```
-export AWS_CLUSTER_NAME=eksworkshop-eksctl
-export KFAPP=${AWS_CLUSTER_NAME}
+mkdir -p ${KF_DIR}
+cd ${KF_DIR}
+kfctl build -V -f ${CONFIG_URI}
 ```
-
-Initialize the cluster:
+Set an environment variable pointing to your local configuration file
 
 ```
-kfctl init ${KFAPP} --config=${CONFIG} -V
+export CONFIG_FILE=${KF_DIR}/kfctl_aws.0.7.0.yaml
 ```
 
-Create and apply AWS and Kubernetes resources in the cluster:
+Replace EKS Cluster Name, AWS Region and IAM Roles in your $(CONFIG_FILE)
 
 ```
-cd ${KFAPP}
+sed -i -e 's/kubeflow-aws/'"$AWS_CLUSTER_NAME"'/' ${CONFIG_FILE}
+sed -i "s@eksctl-eksworkshop-eksctl-nodegroup-ng-a2-NodeInstanceRole-xxxxxxx@$ROLE_NAME@" ${CONFIG_FILE}
```

Comment:
Do we use single node group or two node groups?

Reply:
I took the defaults from eksworkshop.com and only have one node group